program/rust/src/tests/test_upd_aggregate.rs

```diff
@@ -100,8 +100,6 @@ fn test_upd_aggregate() {
 
         assert_eq!(price_data.agg_.price_, 100);
         assert_eq!(price_data.agg_.conf_, 10);
-        assert_eq!(price_data.twap_.val_, 100);
```

Comment:
Maybe better to keep the asserts and run `c_upd_twap` after `c_upd_aggregate` above

Reply:
hmm we already have a `test_ema.rs` which runs `c_upd_twap` do we need it here again?